src/adapter/activateDaffodilDebug.ts

```diff
@@ -24,6 +24,35 @@ import * as launchWizard from '../launchWizard/launchWizard'
 import * as omegaEditClient from '../omega_edit/client'
 import * as dfdlLang from '../language/dfdl'
 
+// Method to file path for program and data
+async function getFile(fileRequested, label, title) {
+  let file = ''
+
+  if (fileRequested && fs.existsSync(fileRequested)) {
+    file = fileRequested
+  } else if (fileRequested && !fs.existsSync(fileRequested)) {
+    file = ''
+  } else {
```

Comment:
I dont' think I understand why we need this `fileRequested` logic? If a caller passes in `fileRequested` then we don't actually use an open dialog and get the file eslehwere? I think this needs comments explain the parameters and expected return of this function, but it's not clear based on the name.

Reply:
I added  better comment above it, does it better answer why it exists?

Reply:
Yes sounds good :+1: